### PR TITLE
Completed Binary Search-1 - Please review

### PR DESCRIPTION
**Pull request by arzoouzma01**

### CODE REVIEW

File: Problem_1.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+   
+Search in a Rotated Sorted Array (https://leetcode.com/problems/search-in-rotated-sorted-array/)
+*/
+
+//Time comlpexity - O(n)
```
Comment (super30admin) [inline Problem_1.java:6]: o(log(n). Why do think it O(n)?

File: Problem_2.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	static int binarySearch(int arr[], int l, int r, int x) 
+	{ 
+		if (r>=l) 
+		{ 
+			int mid = l + (r - l)/2; 
+			if (arr[mid] == x) 
+				return mid; 
+			if (arr[mid] > x) 
+				return binarySearch(arr, l, mid-1, x); 
+			return binarySearch(arr, mid+1, r, x);
```
Comment (super30admin) [inline Problem_2.java:30]: Avoid recursive binary search. It takes extra space on recursive stack

**Comment by super30admin** [comment]
Kindly review inline comments

**Comment by arzoouzma01** [comment]
closing request

**Pull request closed without merging**